src/components/Providers.tsx

```diff
@@ -85,12 +93,63 @@ const { provider, webSocketProvider } = configureChains(
   ],
 );
 
+const projectId = process.env.NEXT_PUBLIC_WALLETCONNECT_PROJECT_ID!;
+
+if (!projectId) {
+  console.error('NEXT_PUBLIC_WALLETCONNECT_PROJECT_ID variable missing.');
+}
+
+const appInfo = {
+  appName: 'Bridge to Arbitrum',
+  projectId,
+};
+
+const { wallets } = getDefaultWallets({
+  ...appInfo,
+  chains,
+});
+
+const connectors = connectorsForWallets([
+  ...wallets,
+  {
+    groupName: 'More',
+    wallets: [trustWallet({ chains, projectId })],
+  },
+]);
+
 const client = createClient({
   autoConnect: true,
+  connectors,
   provider,
-  webSocketProvider,
 });
 
-export const WagmiProvider = ({ children }: { children: ReactNode }) => {
+const WagmiProvider = ({ children }: { children: ReactNode }) => {
   return <WagmiConfig client={client}>{children}</WagmiConfig>;
 };
+
+const theme = darkTheme();
+const rainbowkitTheme = {
+  ...theme,
+  colors: {
+    ...theme.colors,
+    accentColor: 'var(--blue)',
+  },
+  fonts: {
+    ...theme.fonts,
+    body: "'Space Grotesk', sans-serif",
+  },
+};
+
+export function Providers({ children }: PropsWithChildren) {
+  return (
+    <WagmiProvider>
+      <RainbowKitProvider
```

Comment:
rainbowkit adds a `div` here so the CSS is messed up

input field on main page is so short
<img width="536" alt="image" src="https://github.com/OffchainLabs/retryable-tx-panel/assets/13184582/264c169f-cd01-4f36-9cc5-85c049de7ffe">

Reply:
Fixed